index.bs

```diff
@@ -65,13 +65,13 @@ url: https://aomediacodec.github.io/av1-spec/av1-spec.pdf#page=121; spec: AV1; t
 url: https://aomediacodec.github.io/av1-spec/av1-spec.pdf#page=46; spec: AV1; type: dfn;
 	text: Metadata OBU
 
-url: https://aomediacodec.github.io/av1-spec/av1-spec.pdf#page=121; spec: AV1; type: dfn;
+url: https://aomediacodec.github.io/av1-spec/av1-spec.pdf#page=49; spec: AV1; type: dfn;
 	text: Redundant Frame Header OBU
 
-url: https://aomediacodec.github.io/av1-spec/av1-spec.pdf#page=121; spec: AV1; type: dfn;
```

Comment:
The `page=121` part was apparently intended to point to Section 6.6 (Padding OBU semantics) in the AV1 spec. Since all the other xxx OBU links point to the Syntax rather than the Semantics sections, I fixed this page number by pointing to Section 5.7 (Padding OBU syntax).